introduction.md

```diff
@@ -1,2 +1,9 @@
 My name is Paul Stalenhoef, I live in the Netherlands.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?